raft.go

```diff
@@ -748,7 +750,10 @@ func (r *Raft) runLeader() {
 
 	// Setup leader state
 	r.leaderState.commitCh = make(chan struct{}, 1)
-	r.leaderState.inflight = newInflight(r.leaderState.commitCh)
+	r.leaderState.commitment = newCommitment(r.leaderState.commitCh,
+		append([]string{r.localAddr}, r.peers...),
+		r.getLastIndex()+1)
```

Comment:
Ok talked though this with @sean- and it makes sense now - this will be the first index that can be committed _under this term_. This is probably worth a comment here as well because it's super important to how this operates (I see that it is documented over in commitment.go).